books/architecture/src/resources/sockets/ports.md

```diff
@@ -0,0 +1,2 @@
+# Index of ports
+This is an index of all of the network sockets Cuprate actively uses.
```

Comment:
Don't think we open any right now, right? Maybe networking ones in the alpha binary?

Reply:
Yeah not yet, we would need #176. For the initial alpha binary I would be ok with not accepting inbound connections